ms-common-auth/src/main/java/net/trajano/ms/auth/internal/HttpAuthorizationHeaders.java

```diff
@@ -27,7 +25,7 @@
      *            authorization
      * @return an array consisting of the user name and password as strings.
      */
-    public static String[] parseBasicAuthorization(@NotNull final String authorization) {
+    public static String[] parseBasicAuthorization(String authorization) {
 
         if (authorization == null) {
             return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty array instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)